### DISCUSSION TITLE
How to include files like .txt and .yaml in the obfuscated module?

### DISCUSSION THREAD

**Discussion by willEternal8** [dashingsoft/pyarmor#1691] [Q&A] [answered]
When we run `pyarmor gen  -r some_module`  only files with extension in `pyexts` are included. I want to include some other files like `requirements.txt` and `logging.yaml` in the obfuscated code. Is there a way to accomplish this by changing some of the configurations in `pyarmor 8.4.7`.

In previous versions we used to accomplish by adding `"manifest": include some_module/requirements.txt` in the config file.

**Comment by jondy** [comment]
Refer to https://pyarmor.readthedocs.io/en/latest/tutorial/obfuscation.html#copying-package-data-files

**Comment by khadeom** [comment]
Hi @jondy, If we want to have all the data files we can use `pyarmor cfg data_files=*` but what if i only want to have few specific files for example `Dockerfile`, `env` and `*.yaml`. What will be the configuration for this. I have tried `pyarmor gen data_files=Dockerfile,*.yaml,env` but this did not work.  

Current I am copying the required files after obfuscation

**Comment by jondy** [comment, marked as answer]
Using whitespace as separator, for example, `pyarmor gen data_files="Dockerfile *.yaml env"`

**Reply by willEternal8** [reply]
Thanks
